client/packages/coldchain/src/Equipment/ImportAsset/ImportReviewDataTable.tsx

```diff
@@ -29,24 +31,12 @@ export const ImportReviewDataTable: FC<ImportReviewDataTableProps> = ({
     offset: 0,
   });
   const [searchString, setSearchString] = useState<string>(() => '');
-  const columnDescriptions: ColumnDescription<ImportRow>[] = [
-    {
-      key: 'assetNumber',
-      width: 70,
-      sortable: false,
-      label: 'label.asset-number',
-    },
-    {
-      key: 'catalogueItemCode',
-      width: 50,
-      sortable: false,
-      label: 'label.catalogue-item-code',
-    },
-  ];
```

Comment:
Sorry, little bit of jumping around - didn't get why everywhere was a different order